contributing_to_docs/doc_guidelines.adoc

```diff
@@ -139,6 +140,8 @@ separated, global use of `{product-version}` across all branches is probably
 less useful, but it is available if you come across a need for it. Just consider
 how it will render across any branches that the content appears in.
 
+Markup, such as in commands, persists in headings.
```

Comment:
@adellape yeah, it does render strangely. I can revise this to state that we should not include markup in headings. As long as we're consistent, I'm happy! :smiley: